locales/en/messages.json

```diff
@@ -2755,11 +2755,11 @@
     "dataflashButtonSaveFile": {
         "message": "Save flash to file..."
     },
-    "dataflashButtonSaveFileDeprecated": {
-        "message": "Save flash to file... (deprecated)"
+    "dataflashSavetoFileNote": {
+        "message": "Directly saving flash to file is convenient for small files, but is slow and inherently prone to error / file corruption.<br>If this method fails, do not log a support request - use Mass Storage mode instead."
```

Comment:
Can we please turn this around and put it like:

```
Directly saving flash to file s slow and inherently prone to error / file corruption.<br>In some cases it will work for small files, but this is not supported and support requests for it will be closed without comment.
```

Reply:
@ctzsnooze: Sounds reasonable. Not sure if currently MacOS (which I know you are using) is better at dealing with large volume serial data streams than Windows, making it work for you. But a couple of years back there was a version of MacOS that would simply lock up the USB port every single time it was flooded with serial data by trying to download a log, so 'always works' is just a matter of short memory. ;-)

Reply:
Yes I remember that episode.  It was back in 2017-18.  That was a long time ago.  Since then it has been bullet proof.